### PR TITLE
chore: add PR review app yml file

### PR DESCRIPTION
**Pull request by WilcoFiers**
Closes issue:

## Reviewer checks

**Required fields, to be filled out by PR reviewer(s)**

- [ ] Follows the commit message policy, appropriate for next version
- [ ] Code is reviewed for security

### CODE REVIEW

File: .github/review.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+pending_description: Pull request not yet approved for security and a correct PR title
+pending_summary: >
+  Awaiting security and PR title review. To approve this pull request, please click the "Approve" button above.
+  You may **not** review your own Pull Request.
+  
+approved_description: Pull request has been reviewed and approved for security and a correct PR title
+approved_summary: Pull request has been approved for security and having an incorrect PR title by @${approver}.
+
+declined_description: Pull request contains security concerns or has an incorrect PR title
+declined_summary: Commit was reviewed and declined for security and/or an incorrect PR title by @${approver}.
```
Comment (stephenmathieson) [inline .github/review.yml:16]: No need to include this here, this is all the "default" text.

**Review by stephenmathieson — COMMENTED** [review]
LGTM but you can remove the majority of the file 😄

File: .github/review.yml (hunk 1 above)
Comment (WilcoFiers) [reply .github/review.yml:16]: Doubt it, I customised these.

Comment (stephenmathieson) [reply .github/review.yml:16]: Yes you did. I'm sorry. 🤦

File: .github/review.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Require that merge commits have security review performed on them.
+review_merges: true
+
+approve_button_tooltip: No security or PR title concerns
+decline_button_tooltip: Found security or PR title  concerns
```
Comment (straker) [inline .github/review.yml:5]:
```suggestion
decline_button_tooltip: Found security or PR title concerns
```

**Review by straker — APPROVED** [review]

**Comment by stephenmathieson** [comment]
Should we remove the PR checkboxes in this PR?

**Comment by stephenmathieson** [comment]
If merged, we should make "Security Review" a **required** check.

**Review by straker — APPROVED** [review]

**Review by straker — APPROVED** [review]

**Pull request merged**